manifests/sar_metric.pp

```diff
@@ -0,0 +1,60 @@
+define puppet_metrics_collector::sar_metric (
+  String                    $output_dir,
+  String                    $scripts_dir,
+  Enum['absent', 'present'] $metric_ensure = 'present',
+  String                    $metrics_type = $title,
+  String                    $cron_minute = '*/5',
+  Integer                   $retention_days = 90,
+  Integer                   $polling_frequency_seconds = 1,
+  Integer                   $collection_frequency = 5, #minutes
+  String                    $metric_script_file = 'generate_system_metrics',
+) {
+
+  $metrics_output_dir = "${output_dir}/${metrics_type}"
```

Comment:
The output directory is inconsistent with https://github.com/puppetlabs/puppetlabs-puppet_metrics_collector/blob/master/files/generate_system_metrics#L144, which breaks the tidy scripts and adds a few additional directories. 

generate_system_metrics has `system_${metric_type}` and this creates `${metric_type}`

```
# ls -d *cpu *memory
cpu  memory  system_cpu  system_memory
```

Reply:
I changed generate_system_metrics to not add the "system_" to the front, and then changed the declarations to put system_ in the name, that makes it more like pe_metric and should fix this.

Reply:
👍